webpack.config.js

```diff
@@ -54,6 +54,15 @@ module.exports = (env, argv) => {
                         enforce: true,
                         // Do not add `chunks: 'all'` here because you'll break the app entry point.
                     },
+                    vendor: {
+                        test: /[\\/]node_modules[\\/]/,
+                        name: 'vendor',
+                        enforce: true,
+                        chunks (chunk) {
```

Comment:
this looks like it was supposed to be a property:
```suggestion
                        chunks: (chunk) => {
```